test/FieldArray.test.tsx

```diff
@@ -37,6 +37,26 @@ describe('<FieldArray />', () => {
     );
   });
 
+  it('renders component and passes exra props down', () => {
```

Comment:
Just a little misspelling.
```suggestion
  it('renders component and passes extra props down', () => {
```

Reply:
good catch 👍